### DISCUSSION TITLE
Changing the developement server IP and port when testing the app

### DISCUSSION THREAD

**Discussion by Ubatami** [RocketChat/Rocket.Chat.ReactNative#2881] [Q&A] [answered]
Hi, 
I am sorry it might be dumb question but i am complete newby in react developement.
I wanted to know when running the app for testing in an emulator, i am getting the message
connecting to the development server like in the image bellow
![image](https://user-images.githubusercontent.com/68062240/107195630-dfefc680-69f1-11eb-94a2-5c185c9ad092.png)
and the app is try to connect to 10.0.2.2:8081
![image](https://user-images.githubusercontent.com/68062240/107195827-234a3500-69f2-11eb-9021-639be785c19a.png)

Could someone tell me where and how to change this ip and port to the developement server and can I use some other production/public RocketChat server for testing ?

Thanks.

**Comment by diegolmello** [comment, marked as answer]
Hey.
`10.0.2.2:8081` is not a Rocket.Chat server.
The app is loading the JS bundle (React part of the app).
I recommend you to read https://reactnative.dev/docs/getting-started

**Reply by Ubatami** [reply]
Thank you for your help, you were right.
Going to the developer menu --> settings --> Debug server host and port setting and setting the localip and 8081 allowed application to load.
![image](https://user-images.githubusercontent.com/68062240/107225143-bc8c4200-6a18-11eb-85a9-c32ae2d33ea3.png)
![image](https://user-images.githubusercontent.com/68062240/107225354-f9f0cf80-6a18-11eb-8b8c-4d86cb866f11.png)

Thanks.
